deploy/Chart/values.yaml

```diff
@@ -133,3 +133,25 @@ bicep:
   image: ghcr.io/radius-project/bicep
   # Default tag uses Chart AppVersion.
   # tag: latest
+
+preupgrade:
+  # Enable pre-upgrade preflight checks
+  enabled: true
+  # Container image for pre-upgrade checks
+  image: ghcr.io/radius-project/pre-upgrade
+  # Default tag uses Chart AppVersion.
+  # tag: latest
+  # Individual preflight checks configuration
+  checks:
+    # Version compatibility check
+    version: true
```

Comment:
for now, just set the `version` preflight check. we can do more in the future if we want.